### PR TITLE
Added scroll options, nav bar updation and also social icons

### LINKED ISSUE #7: Added scroll down button in header
image

some thing looks like the below one

https://srisatyalokesh.github.io/

### LINKED ISSUE #11: Icons at top
The icons on top in phone view are coming to 2 rows, the icons are sticky, need some space
 between them. (Line space)

### PR DESCRIPTION
**Pull request by VasaviLagishetty**
scroll_down
scroll_up

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -1,5 +1,20 @@
 
 <!-- Intro Start -->
+<style>
+#icon{
+  position: absolute;
+	bottom:25px;
```
Comment (SriSatyaLokesh) [inline _includes/header.html:6]: is this fine in mobile view too?

**Review by SriSatyaLokesh — APPROVED** [review]

File: _includes/header.html (hunk 1 above)
Comment (VasaviLagishetty) [reply _includes/header.html:6]: yeah
mobile_view_scroll_down

Comment (SriSatyaLokesh) [reply _includes/header.html:6]: That's great
pull latest changes
then push again

Comment (SriSatyaLokesh) [reply _includes/header.html:6]: `git pull origin master` 
then push

**Pull request merged**